Makefile

```diff
@@ -798,6 +799,7 @@ deploy/service: deploy/envoy deploy/route
 		-p STRIMZI_OLM_PACKAGE_NAME="${STRIMZI_OLM_PACKAGE_NAME}" \
 		-p KAS_FLEETSHARD_OLM_PACKAGE_NAME="${KAS_FLEETSHARD_OLM_PACKAGE_NAME}" \
 		-p CLUSTER_LIST="${CLUSTER_LIST}" \
+		-p SUPPORTED_CLOUD_PROVIDERS="${SUPPORTED_CLOUD_PROVIDERS}" \
```

Comment:
Can we add a line or two on the https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/539b3ee14b821c3b642eac549eda47de01a97d4b/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager docs about this variable and what the default is?

Reply:
👍